esmvalcore/cmor/_fixes/fix.py

```diff
@@ -197,21 +218,38 @@ def get_fixes(project, dataset, mip, short_name, extra_facets=None):
         return fixes
 
     @staticmethod
-    def get_fixed_filepath(output_dir, filepath):
+    def get_fixed_filepath(
+        output_dir: str | Path,
+        filepath: str | Path,
+        create_temporary_dir: bool = False,
```

Comment:
Good that you're preserving backward compatibility here. How about calling the argument `add_suffix` or `add_unique_suffix` or something similar? And then explain this in the help text with e.g. 'adds a unique suffix to output dir for thread safety'? To me, that describes the feature better than `create_temporary_dir`, because the directory isn't really temporary in the sense that it is not stored in the location for temporary files nor is it removed automatically afterward.